docs/read-side.md

```diff
@@ -272,7 +271,7 @@ We recommend that you store Read Model data in a denormalized form so that your
 
 A projection function is used to accumulate the event data in a **Read Model storage**. Each projection function takes the storage object and event information. The event information includes the aggregateID, timestamp and payload.
```

Comment:
```suggestion
A projection function is used to accumulate the event data in **Read Model storage**. Each projection function takes the storage object and event information. The event information includes the aggregateID, timestamp, and payload.
```

Reply:
"takes the storage object and event information" - takes? Why 'takes'?

Reply:
Changed to 'receives'